chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
-are pretty bad.\sidenote{Gray Kimbrough's
-\textit{Uncluttered Stata Graphs} code is an excellent
-default replacement that is easy to install.
-\url{https://graykimbrough.github.io/uncluttered-stata-graphs/}}
-Thankfully, graphics tools like Stata are highly customizable.
-There is a fair amount of learning curve associated with
-extremely-fine-grained adjustment,
-but it is well worth reviewing the graphics manual
-to understand how to apply basic colors, alignments, and labelling tools\sidenote{
-\url{https://www.stata.com/manuals/g.pdf}}
-The manual is worth skimming in full, as it provides
-many visual examples and corresponding code examples
-that detail how to produce them.
-
-Graphics are hard to get right because you, as the analyst,
-will already have a pretty good idea of what you are trying to convey.
-Someone seeing the illustration for the first time,
-by contrast, will have no idea what they are looking at.
-Therefore, a visual image has to be compelling in the sense
-that it shows that there is something interesting going on
-and compels the reader to figure out exactly what it is.
-A variety of resources can help you
-figure out how best to structure a given visual.
-The \textbf{Stata Visual Library}\sidenote{
-\url{https://worldbank.github.io/Stata-IE-Visual-Library/}}
- has many examples of figures that communicate effectively.
+To accomplish this, you will need to make sure that you have an effective data management system, including naming, file organization, and version control.
+Just like you did with each of the analysis datasets, name each of the individual analysis files descriptively.
+Code files such as \path{spatial-diff-in-diff.do}, \path{matching-villages.do}, and \path{summary-statistics.do} 
+are clear indicators of what each file is doing, and allow you to find code quickly.
+If you intend to numerically order the code as they appear in a paper or report, 
+leave this to near publication time.
+
+% Self-promotion ------------------------------------------------
+Our team has created a few products to automate common outputs and save you 
+precious research time.
+The \texttt{ietoolkit} package includes two commands to export nicely formatted tables.
+\texttt{iebaltab}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Iebaltab}} creates and exports balance tables to excel and {\TeX}. 
+\texttt{ieddtab}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Ieddtab}} does the same for difference-in-differences regressions.
+The \textbf{Stata Visual Library}\sidenote{	\url{https://worldbank.github.io/Stata-IE-Visual-Library/}}
+has examples of graphs created in Stata and curated by us.\sidenote{A similar resource for R is \textit{The R Graph Gallery}. \\\url{https://www.r-graph-gallery.com/}}
+\textbf{Data visualization}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_visualization}} \index{data visualization}
+is increasingly popular, but a great deal lacks in quality.\cite{healy2018data,wilke2019fundamentals}
```

Comment:
This sentence is odd. What are we trying to say exactly?

Reply:
people make crap graphs

Reply:
I got the last part of the sentence. But isn't "_Data visualization is increasingly popular_" a weird statement? I agree with what I understand that you are trying to say, but it seems like something one say in passing and not writing in a book. And data visualization has been very poplar for a long time, but there is a boom in it and it is turning into a field of its own. No strong feelings about this if you all think this is good.

Reply:
I don't have strong feelings about this sentence, for one side or the other. I also don't have a better suggestion of phrasing, but we can brainstorm about it during the meeting.

Reply:
removed in 3a9f1d8